charts/drone/values.yaml

```diff
@@ -48,7 +48,8 @@ ingress:
   hosts:
     - host: chart-example.local
       paths:
-        - "/"
+        - path: "/*"
```

Comment:
I think wildcards are not supported in ingress paths. At least they don't work with my traefik ingress controller and the kubernetes [documentation](https://kubernetes.io/docs/concepts/services-networking/ingress/#examples) doesn't seem to mention them.

```yaml
- path: /
  pathType: Prefix
```
... should basically do the same thing and a lot of charts use it as suggested setting.

Everything else works fine (k3s v1.22.7)! Thanks for updating the chart, @jimsheldon :100: :1st_place_medal:

Reply:
Thanks for the feedback @pauvos! I'm not sure what documentation I was reading, but I agree that `pathType: Prefix` makes more sense as a default. I have pushed a commit with this change, which also adds some comments.